app/src/components/ResourceInfoCard.tsx

```diff
@@ -39,6 +39,8 @@ export default function ResourceInfoCard({ resourceInfo, authoring }: ResourceIn
   const ctx = trpc.useContext();
   const [isConfirmationModalOpen, setIsConfirmationModalOpen] = useState(false);
 
+  authoring = authoring === undefined ? false : true;
+
```

Comment:
Rather than doing this here, if you look at where the `ResourceCards` component is called in `src/pages/authoring/[resourceType]/index.tsx`, you will see that it is called with `authoring={true}`. You can do this where `ResourceCards` is rendered in `src/pages/[resourceType]` and `src/pages/[resourceType]/search-result.tsx` with `authoring={false}`

Reply:
I agree with Elsa's approach - if we set `authoring` to true or false when passing it into `ResourceInfoCard`, then we can avoid having to reassign the value of `authoring`, which is preferred 👍

Reply:
Great idea!